external/WICED/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
```

Comment:
Please use new sentence for this as I guided.

Reply:
I will use below setence. 

+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt

BR, HJ Lee.